client/src/components/molecules/CommandPallete/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { Search } from 'react-feather'
+import { useRouter } from 'next/router'
+import { Combobox, Dialog, Transition } from '@headlessui/react'
+import React, { FC, useState, useEffect, Fragment } from 'react'
+
+import { IProject } from '~/shared/interfaces'
+
+type Props = {
+  isOpen: boolean
+  closeModal: () => void
+  projects: IProject[]
+}
+
+const CommandPallete: FC<Props> = ({ isOpen, closeModal, projects }): JSX.Element => {
+  const router = useRouter()
+  const [query, setQuery] = useState<string>('')
```

Comment:
Instead of passing `projects` as props, you can call the `projects` state directly from the redux unto the component itself. With this we can avoid multiple props to be passed on to the component